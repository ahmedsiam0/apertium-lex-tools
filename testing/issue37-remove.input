^a<a>/removemefirst<b>/thenremoveme<c>/keepme<d>$
^x<x>/removemeonly<b>/keepme<c>/keepme<d>$
