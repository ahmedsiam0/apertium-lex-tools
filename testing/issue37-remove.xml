<rules>
  <rule weight="3.14159">
    <match lemma="a" tags="a"><remove lemma="removemefirst"/></match>
  </rule>
  <rule weight="2.71828">
    <match lemma="a" tags="a"><remove lemma="thenremoveme"/></match>
  </rule>

  <rule weight="1.61803">
    <match lemma="x" tags="x"><remove lemma="removemeonly"/></match>
  </rule>
</rules>
