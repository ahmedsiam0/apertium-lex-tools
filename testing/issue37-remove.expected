^a<a>/keepme<d>$
^x<x>/keepme<c>/keepme<d>$
